yt/frontends/boxlib/data_structures.py

```diff
@@ -1128,16 +1132,6 @@ def _parse_parameter_file(self):
                     self.parameters[fields[0]] = fields[1].strip()
                 line = next(f)
 
-            # runtime parameters that we overrode follow "Inputs File
```

Comment:
it's not clear to me why this can be removed

Reply:
This is doing the same thing as `_parse_cparams()` in BoxlibDataset, but doesn't strip the trailing space from `p` or run `v` through `_guess_pcast()`.